dev/CommonStyles/MediaTransportControls_themeresources.xaml

```diff
@@ -24,14 +24,16 @@
         </ResourceDictionary>
         <ResourceDictionary x:Key="Light">
             <Thickness x:Key="MediaTransportControlsTitleSafeBounds">48,0,48,27</Thickness>
-            <StaticResource x:Key="MediaTransportControlsPanelBackground" ResourceKey="ControlOnImageFillColorDefaultBrush" />
+            <StaticResource x:Key="MediaTransportControlsPanelBackground" ResourceKey="AcrylicInAppFillColorDefaultBrush" />
             <StaticResource x:Key="MediaTransportControlsFlyoutBackground" ResourceKey="AcrylicBackgroundFillColorDefaultBrush" />
             <StaticResource x:Key="MediaTransportControlsThumbBorderBackground" ResourceKey="ControlStrongFillColorDefaultBrush" />
             <StaticResource x:Key="MediaTransportControlsFillTimeElapsedText" ResourceKey="TextFillColorSecondaryBrush" />
             <StaticResource x:Key="MediaTransportControlsFillMediaText" ResourceKey="TextFillColorPrimaryBrush" />
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
+    <x:Double x:Key="MediaTransportControlsAppBarButtonSize">40</x:Double>
```

Comment:
>MediaTransportControlsAppBarButtonSize [](http://example.com/codeflow?start=21&length=38)

This change has some compatability concerns.  If a developer was overriding the MTCMediaButtonWidth and MTCMediaButtonHeight resources to customize their MTC then this change would stop that from working.  I would suggest instead we provide a Winui2 value for MTCMediaButtonWidth and MTCMediaButtonHeight in the ThemeDictionaries above.